test/ScanDependencies/binary_framework_dependency.swift

```diff
@@ -17,6 +17,7 @@ import Foo
 
 // CHECK: "swiftPrebuiltExternal": "Foo"
 // CHECK:      "swiftPrebuiltExternal": "Foo"
+// CHECK:      "swiftPrebuiltExternal": "Foo"
```

Comment:
I can't understand this test anymore. Can you add a bit context around those 3 matches to be clear?

Reply:
Added comments to explain each occurrence. 👍🏼